these-de-medecine-paris-xi.csl

```diff
@@ -0,0 +1,141 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" default-locale="fr-FR">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>These de medecine Paris XI</title>
+    <id>http://www.zotero.org/styles/these-de-medecine-paris-xi</id>
+    <link href="http://www.zotero.org/styles/these-de-medecine-paris-xi" rel="self"/>
+    <link href="http://www.zotero.org/styles/medecine-sciences" rel="template"/>
+    <link href="http://www.zotero.org/styles/plos" rel="template"/>
+    <link href="http://www.medecinesciences.org" rel="documentation"/>
```

Comment:
is the style for Paris XI documented anywhere? We really would like online documentation, especially for styles that don't correspond to any easily accessible publications like journal styles.